src/loop.cpp

```diff
@@ -31,7 +46,7 @@ static int calc_pitch(int note, float frac) {
 	float fine = frac / 2 + 0.5;
 	fine *= .90;
 
-	return sidScale[note - 12 - 1] + (sidScale[note - 12 + 2] - sidScale[note - 12]) * fine;
+	return (sidScale[note - 12 - 1] + (sidScale[note - 12 + 2] - sidScale[note - 12]) * fine);
```

Comment:
why the braces? they're not needed and usually, people don't write them here. (It would make `return` look like a function call which it isn't)

Reply:
ok used to arduino returning stuff with braces...